tensorflow/core/ops/math_ops.cc

```diff
@@ -1193,7 +1193,7 @@ Status SparseSegmentReductionShapeFn(InferenceContext* c) {
   return OkStatus();
 }
 
-Status SparseSegmentReductionGradShapeFn(InferenceContext* c) {
+Status SparseSegmentReductionGradShapeFnImpl(InferenceContext* c, bool v2) {
```

Comment:
Can you give a more meaningful name than `v2`?

Reply:
Done, renamed to `outputs_unique_indices`.